fbw-a32nx/src/systems/fmgc/src/efis/EfisSymbols.ts

```diff
@@ -187,8 +188,13 @@ export class EfisSymbols<T extends number> {
     const vnavPredictionsChanged = this.lastVnavDriverVersion !== this.guidanceController.vnavDriver.version;
     this.lastVnavDriverVersion = this.guidanceController.vnavDriver.version;
 
-    const hasSuitableRunway = (airport: Airport): boolean =>
-      airport.longestRunwayLength >= 1500 && airport.longestRunwaySurfaceType === RunwaySurfaceType.Hard;
+    const hasSuitableRunway = (airport: Airport): boolean => {
+      const minRwyLengthDisplay = parseInt(NXDataStore.get('CONFIG_MIN_RWY_LENGTH', '1500'));
```

Comment:
It would be better to store this in a class property and use NXDataStore.getAndSubscribe rather than parsing a string every time we check a runway (a lot!).

Reply:
Thank you for your suggestion. Definitely trying that to save some resources :)